framework/mesh/mesh_generator/orthogonal_mesh_generator.cc

```diff
@@ -257,37 +254,33 @@ OrthogonalMeshGenerator::CreateUnpartitioned2DOrthoMesh(const std::vector<double
 
         face.neighbor = true;
         if (v == 1 and j != max_j)
-          face.neighbor = cmap[i][j + 1]; /*XMAX*/
+          face.neighbor = cmap[i][j + 1]; /*XMIN*/
         if (v == 3 and j != 0)
-          face.neighbor = cmap[i][j - 1]; /*XMIN*/
+          face.neighbor = cmap[i][j - 1]; /*XMAX*/
         if (v == 2 and i != max_i)
-          face.neighbor = cmap[i + 1][j]; /*YMAX*/
+          face.neighbor = cmap[i + 1][j]; /*YMIN*/
         if (v == 0 and i != 0)
-          face.neighbor = cmap[i - 1][j]; /*YMIN*/
+          face.neighbor = cmap[i - 1][j]; /*YMAX*/
```

Comment:
These are interesting comments, aren't these interior cells here? I wonder why these comments were there in the first place... Either way, they should be reverted. The change in boundary ordering doesn't change the orientation of the cell during construction, especially not interior cells.